config-model/src/test/java/com/yahoo/config/model/MockModelContext.java

```diff
@@ -139,6 +139,9 @@ public Zone zone() {
 
             @Override
             public boolean useFdispatchByDefault() { return true; }
+
+            @Override
+            public boolean useAdaptiveDispatch() { return false; }
```

Comment:
This Properties implementation can probably be replaced by TestProperties...

Reply:
Done